wAbout/Resizing-Form-Windows.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: Resizing Form Text
+title: Resizing Interject Windows
```

Comment:
it might be worth putting a short description and link to this page on the troubleshooting page, in case people go there for help for this issue